code/modules/food/beverages/bottler.dm

```diff
@@ -0,0 +1,448 @@
+
+/obj/machinery/bottler
+	name = "bottler unit"
+	desc = "A machine that combines ingredients and bottles the resulting beverages."
+	icon = 'icons/obj/kitchen.dmi'
+	icon_state = "bottler_off"
+	density = 1
+	anchored = 1
+	var/obj/item/obj_1 = null
+	var/obj/item/obj_2 = null
+	var/obj/item/obj_3 = null
+	var/list/datum/bottler_recipe/available_recipes
+	var/list/acceptable_items
+	var/glass_bottles = 10
+	var/plastic_bottles = 20
+	var/cans = 25
+	var/bottling = 0
+
+/obj/machinery/bottler/New()
+	if(!available_recipes)
+		available_recipes = new
+		acceptable_items = new
+		//These are going to be acceptable even if they aren't in a recipe
+		acceptable_items |= /obj/item/weapon/reagent_containers/food/snacks
+		acceptable_items |= /obj/item/weapon/reagent_containers/food/drinks/cans
+		//the rest is based on what is used in recipes so we don't have people destroying the nuke disc
+		for (var/type in subtypesof(/datum/bottler_recipe))
+			var/datum/bottler_recipe/recipe = new type
+			if(recipe.result) // Ignore recipe subtypes that lack a result
+				available_recipes += recipe
+				acceptable_items |= recipe.obj_1_item
+				acceptable_items |= recipe.obj_2_item
+				acceptable_items |= recipe.obj_3_item
+			else
+				qdel(recipe)
+
+/obj/machinery/bottler/attackby(obj/item/O, mob/user, params)
+	if(iswrench(O))		//This being before the NODROP check allows borgs to (un)wrench bottlers in case they need move them to fix stuff
+		playsound(src, 'sound/items/Ratchet.ogg', 50, 1)
+		if(anchored)
+			anchored = 0
+			to_chat(user, "<span class='alert'>\The [src] can now be moved.</span>")
+		else
+			anchored = 1
+			to_chat(user, "<span class='alert'>\The [src] is now secured.</span>")
+		return 1
+	if(O.flags & NODROP)
+		to_chat(user, "<span class='warning'>\The [O] is stuck to your hand, you can't seem to put it down!</span>")
+		return 0
+	if(is_type_in_list(O,acceptable_items))
+		if(istype(O, /obj/item/weapon/reagent_containers/food/snacks))
+			var/obj/item/weapon/reagent_containers/food/snacks/S = O
+			user.unEquip(S)
+			if(S.reagents && !S.reagents.total_volume)		//This prevents us from using empty foods, should one occur due to some sort of error
+				to_chat(user, "<span class='warning'>\The [S.name] is gone, oh no!</span>")
```

Comment:
`\The [S.name]` is fixing a problem you create for yourself by using `[S.name]` instead of `[S]`, should just be `[S]` without the `\The`. You did the same thing a few times throughout the PR, although a few of them are just an extraneous `\the` that aren't needed.